frontend/src/metabase/query_builder/components/NativeQueryEditor/NativeQueryEditor.tsx

```diff
@@ -12,16 +12,8 @@ import "ace/ace";
 import "ace/ext-language_tools";
 import "ace/ext-searchbox";
 import "ace/mode-sql";
-import "ace/mode-mysql";
-import "ace/mode-pgsql";
-import "ace/mode-sqlserver";
-import "ace/mode-json";
```

Comment:
We still need mode-json, otherwise autocomplete will default to SQL keywords for Mongo and Druid